pkg/applicationserver/linking.go

```diff
@@ -68,6 +69,14 @@ func (as *ApplicationServer) startLinkTask(ctx context.Context, ids ttnpb.Applic
 }
 
 type link struct {
+	// Align for sync/atomic.
```

Comment:
This comment is unclear

Reply:
We use this widely across the codebase. See https://golang.org/pkg/sync/atomic/#pkg-note-BUG.